src/legacy/ui/public/styles/_legacy/components/_sidebar.scss

```diff
@@ -112,8 +112,13 @@
       align-self: center;
     }
   }
+}
 
-  .index-pattern-selection:not(.euiComboBox) {
-    padding: $euiSizeS 0;
-  }
+.indexPattern__container {
+  display: flex;
+  align-items: center;
+}
+
+.indexPattern__triggerButton {
+  @include euiTitle('xs');
```

Comment:
Same situation here with `.lnsInnerIndexPatternDataPanel__triggerButton`, let's add the `line-height` value as well:

```
line-height: $euiSizeXXL;
```

See here:
https://github.com/elastic/kibana/blob/ac0f44e6a50235b8673d1541fec21cfd8ffc548c/x-pack/legacy/plugins/lens/public/indexpattern_plugin/_datapanel.scss#L14-L17